autoload/vim_current_word.vim

```diff
@@ -20,6 +20,11 @@ function! vim_current_word#pre_highlight()
   call vim_current_word#clear_current_word_matches()
   let l:vim_current_word_disabled_in_this_buffer = get(b:, 'vim_current_word_disabled_in_this_buffer', 0)
   if !g:vim_current_word#enabled || index(g:vim_current_word#excluded_filetypes, &filetype) >= 0 || l:vim_current_word_disabled_in_this_buffer | return 0 | endif
```

Comment:
If the version below works the same as your suggestion, then apply the change, please. Remove lines 23-27 added by you after, please.

```suggestion
if !g:vim_current_word#enabled || index(g:vim_current_word#excluded_filetypes, &filetype) >= 0 || l:vim_current_word_disabled_in_this_buffer || **(len(g:vim_current_word#included_filetypes) > 0 && index(g:vim_current_word#included_filetypes, &filetype) == -1)**
  return 0
end
```